libnm-core/nm-setting-tc-config.c

```diff
@@ -1259,6 +1381,11 @@ _qdiscs_to_variant (GPtrArray *qdiscs)
 			g_variant_builder_add (&qdisc_builder, "{sv}", "parent",
 			                       g_variant_new_uint32 (nm_tc_qdisc_get_parent (qdisc)));
 
+			for (y = 0; attrs[y]; y++) {
+				g_variant_builder_add (&qdisc_builder, "{sv}", attrs[y],
+				                       nm_tc_qdisc_get_attribute (qdisc, attrs[y]));
```

Comment:
do we really need to call `nm_tc_qdisc_get_attribute()` here which asserts that the name is non-empty?